lunabot_behavior/escape.py

```diff
@@ -0,0 +1,28 @@
+import rospy
+
+from geometry_msgs.msg import Twist
+
+def main():
```

Comment:
Should this be placed into a class like the other behavior calls?

Reply:
yes